### DISCUSSION TITLE
How can I get the pixels of and image?

### DISCUSSION THREAD

**Discussion by ugoi** [codam-coding-college/MLX42#69] [Q&A] [answered]
Is there an equivalent to mlx_put_pixel but it gets the pixel value like mlx_get_pixel? 
My problem is that I want to shift my Mandelbrot picture to the right and therefore I want to get each pixel and shift it to the right. Then I want to draw the Mandelbrot only for the pixels that got cut off. I have already tried `img->instances[0].x += translation;` but the problem with that is that it shifts the coordiante system. So when I want to draw a new Mandelbrot to the same image the (0, 0) pixel will also be shifted to the right. So I can't draw into the cut off area. What is the solution to my problem?

**Comment by W2Wizard** [comment, marked as answer]
You need to figure this out on your own or ask your peer to the left or to the right.

**Reply by ugoi** [reply]
Bro, I already figured this out a decade ago.

**Reply by W2Wizard** [reply]
Well done :D
